src/components/App.js

```diff
@@ -1,27 +1,23 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
+// import { useSelector } from 'react-redux';
```

Comment:
- [ ] [OPTIONAL] To keep your codebase neat, kindly delete the commented code. Commented code makes our project look messy.